api/config/v1alpha1/validation/validation_test.go

```diff
@@ -485,10 +485,103 @@ func TestValidateEtcdCopyBackupsTaskControllerConfiguration(t *testing.T) {
 			if test.concurrentSync != nil {
 				controllerConfig.ConcurrentSyncs = test.concurrentSync
 			}
-			actualErr := validateEtcdCopyBackupsTaskControllerConfiguration(*controllerConfig, fldPath)
-			g.Expect(len(actualErr)).To(Equal(test.expectedErrors))
+			actualErrList := validateEtcdCopyBackupsTaskControllerConfiguration(*controllerConfig, fldPath)
+			g.Expect(len(actualErrList)).To(Equal(test.expectedErrors))
 			if test.matcher != nil {
-				g.Expect(actualErr).To(test.matcher)
+				g.Expect(actualErrList).To(test.matcher)
+			}
+		})
+	}
+}
+
+func TestValidateEtcdOpsTaskControllerConfiguration(t *testing.T) {
+	tests := []struct {
+		name              string
+		config            *druidconfigv1alpha1.EtcdOpsTaskControllerConfiguration
+		numExpectedErrors int
+		matcher           gomegatypes.GomegaMatcher
+	}{
+		{
+			name:              "should allow default etcdOpsTask controller configuration when it is enabled",
```

Comment:
You have now removed the enabled flag. You should change the name to `should use default EtcdOpsTask controller configuration`